plugins/admin/pkg/command/registry/registry.go

```diff
@@ -52,7 +52,7 @@ func NewPrivateRegistryCmd(p *pkg.AdminParams) *cobra.Command {
 	var privateRegistryCmd = &cobra.Command{
 		Use:   "registry",
 		Short: "Manage registry",
-		Long:  `Manage Service deployment from a registry with credentials`,
+		Long:  `Manage service deployment from a registry with credentials`,
```

Comment:
The `Short` description and `Long` do not agree. One reads “Manage registry” the other reads “Manage service deployment ... “

Reply:
@zhanggbj & @12345lcr Have you any comments about the ```Long``` description? Thanks.

Reply:
@maximilien &@zhanggbj How about changing it to `Manage a registry with credentials for service deployment`?

Reply:
@chaozbj @maximilien 
How about "Manage registry used by Knative service deployment"?

Reply:
Or "Manage registries for Knative service deployment image pulling"?